doc(partner-sdk): update the doc to ignore _fivetran_earliest column (#177)
fivetran/fivetran_sdk: how-to-handle-history-mode-batch-files.md

#### `delete_files`
For the active record (the one that has `_fivetran_active = TRUE`) with a given primary key in the destination, the `_fivetran_active` column value is set to FALSE, and the `_fivetran_end` column value is set to the `_fivetran_end` column value of the record with the same primary key in the batch file.

> 

Reviewer: Does the note belong in the context of `delete_files`?

Author: No, this note applies to all files. If any of these files contains the _fivetran_earliest column, we want the partner to ignore it.

> > Important: The `_fivetran_earliest` column will appear in these files. We recommend ignoring it and not writing it to the destination table.

Reviewer 2: writing it to the destination table.
Is writng the correct word? Isn't insert the word used everywhere else

Author: Replace with insert.

> > Important: If the `_fivetran_earliest` column appears in any of the files, we recommend ignoring it and not writing it to the destination table.

Reviewer 2: Why if? It is certain it will be present right?

Author: rephrased it.

> > **Note:** If the `_fivetran_earliest` column appears in any of the files, the column should be ignored and excluded from processing. This column should not be written to the destination table.

Reviewer: ```suggestion
> Important: If the `_fivetran_earliest` column appears in any of the files, ignore it and do not write it to the destination table.
```

Author: Updated

> > **Note:** If the `_fivetran_earliest` column appears in any of the files, the column should be ignored and excluded from processing. This column should not be written to the destination table.

Reviewer: What happens if they end up writing it to the destination table?

Author: Currently, our partners are writing this column to the destination table. We need to fix this issue on our end. Until then, we want our partners to ignore it.

If they do not ignore it and continue writing it to the destination table, it will not be useful to the customer. This is an internal column and should only be used by Fivetran.
